ehr_billing/src/org/labkey/ehr_billing/EHR_BillingContainerListener.java

```diff
@@ -93,6 +93,10 @@ public void containerDeleted(Container c, User user)
             if (chargeableItemCategoriesTable.getTableType() == DatabaseTableType.TABLE)
                 Table.delete(chargeableItemCategoriesTable, containerFilter);
 
+            TableInfo procedurequerychargeidassocTable = EHR_BillingSchema.getInstance().getProcedureQueryChargeIdAssoc();
+            if (procedurequerychargeidassocTable.getTableType() == DatabaseTableType.TABLE)
```

Comment:
I know code for other tables follows this same pattern, but why do we need this check? If the table's not there, we're in a very, very bad state and we don't do similar checks anywhere else.

Reply:
good point, now updated